src/drive/web/modules/public/PublicToolbarByLink.jsx

```diff
@@ -46,12 +45,7 @@ const PublicToolbarByLink = ({
         displayedFolder={displayedFolder}
         isSelectionBarVisible={isSelectionBarVisible}
       >
-        {!isMobile && (
-          <>
-            {hasWriteAccess && <AddButton />}
-            {files.length > 0 && <DownloadFilesButton files={files} />}
-          </>
-        )}
+        {!isMobile && <>{hasWriteAccess && <AddButton />}</>}
```

Comment:
If think we can simplify the condition by removing the fragment : 
```jsx
{!isMobile && hasWriteAccess ?  <AddButton /> : null}
```

Reply:
Est-ce qu'on affiche ce bouton de téléchargement aussi pour un dossier ? Si oui, comment on télécharge un dossier maintenant ? 

On peut avoir un peu plus de contexte dans le commit sur ce que tu essayes de résoudre comme souci ?

Reply:
Vu en daily ce matin, on annule cette modification.